examples/python/extract-logs-write-to-cloud-logging/README.md

```diff
@@ -0,0 +1,66 @@
+## Overview
+
+A Python script that extracts Looker system/audit logs from [System Activity](https://docs.looker.com/admin-options/system-activity) and exports the Logs to Cloud Logging. This example tries to format the output logs like a [GCP Audit Log](https://cloud.google.com/logging/docs/audit/understanding-audit-logs) as best as possible. See [mapping](#gcp-audit-log-fields-to-looker-system-activity-mapping) for comparison between Looker System Activity Fields and GCP Audit Log Fields
+
+> **_NOTE:_**  The script extracts System Activity data from the last 10 minutes. You can then schedule this script to run every 10 minutes using a cron job or equivalent
+
+## Requirements
+- Looker Instance in which you have Admin or `see_system_activity` permission
+- Google Cloud Project with Cloud Logging API enabled
+- [pyenv](https://github.com/pyenv/pyenv#installation) installed
+
+## Deployment
+
+- Clone the repo and navigate to this directory
+  ```
+  git clone https://github.com/looker-open-source/sdk-codegen.git
+  cd sdk-codegen/examples/python/extract-logs-write-to-cloud-logging
+  ```
+
+- Setup Python Virtual environment 
+  ```
+  pyenv install 3.8.2
+  pyenv local 3.8.2
+  python -m venv .venv
+  ```
+
+- Install dependencies 
+  ```
+  pip install looker-sdk
+  pip install --upgrade google-cloud-logging
```

Comment:
Does the `--upgrade` assume the developer already has `google-cloud-logging` installed?

Reply:
This is a mistake. `--upgrade` option is not required.